### DISCUSSION TITLE
Creating and assigning local coordinate in pyAEDT

### DISCUSSION THREAD

**Discussion by Somayaji1995** [ansys/pyaedt#4931] [Q&A] [answered]
I have a PCB which is inclined to Cartesian coordinate, I need to assign orthotropic thermal conductivity for the same for which i have to create one local coordinate system and assign the values. Can you let me know how can I don that ?

**Comment by fwlfHS** [comment, marked as answer]
Have you checked the [reference](https://aedt.docs.pyansys.com/version/stable/API/_autosummary/pyaedt.modeler.modeler3d.Modeler3D.create_coordinate_system.html)?
